src/scss/global.scss

```diff
@@ -296,6 +296,13 @@ textarea {
   }
 }
 
+.custom-select {
+  background-color: rgba($white, 0.12);
+  color: $white;
+  margin-bottom: 0.8em;
+  border-radius: 22px;
+}
```

Comment:
I expect this relates to the `custom` prop on `Form.Control` in `src/pages/Admin/Room/Edit/RoomEditModal.tsx`/etc?

What defines this class name? Are we able to change it?

While this seems like a reasonable use case for including in `global.scss`, we're trying to minimize/reduce what is included here to make things more modular/understandable/etc.

It might be nice to add a comment explaining it's intent/where it applies/etc (eg. to `react-bootstrap`'s `Form.Control` when the `custom` prop is set) to make it easier for future engineers to know why it exists/etc.

Reply:
Yeah, makes sense. I'll add the comment.

The `custom` class name can be changed using the `bsCustomPrefix` prop. By default it uses Bootstrap's custom class name for select.
https://react-bootstrap.github.io/components/forms/#form-control-props

Reply:
Makes sense. Are we intending to have this change applied to all `select`s across the app? Or is it intended for the specific changes being made here/in a generic control?

If the latter, it'd probably be good to give it a more 'localised' name, aligned to BEM patterns on that component:

- http://getbem.com/naming/